frontend/public/components/markdown-view.tsx

```diff
@@ -228,7 +228,7 @@ const IFrameMarkdownView: React.FC<InnerSyncMarkdownProps> = ({
     padding-top: 0;
   }
   </style>
-  <body class="pf-m-redhat-font"><div style="overflow-y: auto;">${markup}</div></body>`;
+  <body class="pf-m-redhat-font pf-c-content"><div style="overflow-y: auto;">${markup}</div></body>`;
```

Comment:
```suggestion
  <body class="pf-m-redhat-font"><TextContent style="overflow-y: auto;">${markup}</TextContent></body>`;
```
instead to maintain dependency chain (assuming you can combine the `style` from the existing `<div>`).

Reply:
Using the `<TextContent>` component within this const doesn't work. 
```suggestion
    const contents = `
  ${linkRefs}
  <style type="text/css">
  body {
    background-color: transparent !important;
    color: ${isEmpty ? '#999' : '#333'};
    font-family: var(--pf-global--FontFamily--sans-serif);
    min-width: auto !important;
  }
  table {
    display: block;
    margin-bottom: 11.5px;
    overflow-x: auto;
  }
  td,
  th {
    border-bottom: 1px solid #ededed;
    padding: 10px;
    vertical-align: top;
  }
  th {
    padding-top: 0;
  }
  </style>
  <body class="pf-m-redhat-font pf-c-content"><TextContent style={{ overflowY: 'auto' }}>${markup}</TextContent></body>`;
```

Reply:
Ah, yes.  My bad.  I should have noticed that is a template literal.  It's ok to leave this one straight HTML with the classname then.